snapcraft.yaml

```diff
@@ -1382,9 +1381,20 @@ parts:
         sed -i 's#libdir=/usr#libdir=${prefix}#' $PC
         sed -i 's#includedir=/usr#includedir=${prefix}#' $PC
       done
-
+  libicuc:
+    source: https://github.com/unicode-org/icu.git
+    source-tag: release-73-1
```

Comment:
I am not able to understand this.. can you explain me this a little bit more?
should it be like this?
# ext:updatesnap
#   version-format:
#     format: 'release-%M-%m'
Major 73 and minor 1?

Reply:
Ops... sorry, I have to add some documentation about this. We use internally some tools to automagically track new versions of each part; but since each project uses its own format for the tags, we need a way of specifying how to parse them. By default, if the format is "MAJOR.MINOR.REVISION" or "vMAJOR.MINOR.REVISION", it can be managed automatically; but for more exotic formats, like in this case, we have to specify how to parse the tags. In this case, that says that the tags are in "release-MAJOR-MINOR" format.

Also, since snapcraft would bark if we just add those extra elements, we add them as comments, and our tool search for a comment that begins with `# ext:updatesnap`, and replaces the '#' in the next lines with a blank space, thus "transforming" that into standard YAML code that can be then processed by us.

Reply:
I have added this also.